### PR TITLE
Add Drupal-with-Solr 5 example.

### PR DESCRIPTION
**Pull request by damontgomery**
As discussed in https://github.com/drud/ddev/pull/2785.

### CODE REVIEW

**Comment by damontgomery** [comment]
Thanks, @rfay. I think this is ready for review.

**Comment by rfay** [comment]
Please add an entry in the top-level README, thanks.

**Comment by damontgomery** [comment]
Link added.

**Review by mikemiles86 — APPROVED** [review]
These changes are clear and straightforward. Using them I was able to successfully get an older version of SOLR setup for my ddev project.

File: docker-compose-services/solr-5/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Solr 5 integration with DDEV
+
+The Solr integration with DDEV assumes a newer version of Solr.
```
Comment (rfay) [inline docker-compose-services/solr-5/README.md:3]:
```suggestion
The documented ApacheSolr integration with DDEV assumes a newer version of Solr.
```

File: docker-compose-services/solr-5/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Limitations
+
+DDEV has a note that it will clean up volumes that match the service name. Since we have two volumes, they can't both match the service name, so that probably won't happen and you'll have to do that manually if it's a concern for you. DDEV creates standard Docker volumes, so they can be deleted with Docker by following standard documentation.
+
+`docker volume ls` to list the volumes
+
+`docker volume rm [volume name]` to remove a volume
+
+You'll see two named something like `ddev-example_solr_opt` and `ddev-example_solr_var`.
```
Comment (rfay) [inline docker-compose-services/solr-5/README.md:46]:
```suggestion
You'll see two named something like `ddev-example_solr_opt` and `ddev-example_solr_var`.

Contributed by [@damontgomery](https://github.com/damontgomery)
```

**Review by rfay — APPROVED** [review]
Thanks for this. I added a couple of small comments

**Comment by damontgomery** [comment]
Thanks again @rfay and @mikemiles86.

**Comment by rfay** [comment]
Thank to you!

**Pull request merged**